ixwebsocket/IXWebSocketTransport.h

```diff
@@ -155,14 +157,25 @@ namespace ix
         std::atomic<bool> _requestInitCancellation;
 
         // Optional Heartbeat
+        std::thread _thread;
```

Comment:
better to rename `_thread` to for example `_heartbeatWatchdogThread`

Reply:
agree